### DISCUSSION TITLE
【PDF嵌入】pdf无法显示或打开 Failed to load PDF file

### DISCUSSION THREAD

**Discussion by fdg513** [tangly1024/NotionNext#1981] [Q&A]
**描述bug**
notion不论是选上传文件还是上传pdf，上传pdf都无法显示或打开

**复现步骤**
出现这个bug的操作步骤
<img width="247" alt="截屏2023-09-03 18 29 07" src="https://github.com/tangly1024/NotionNext/assets/123785991/0f32ab32-d9b2-4138-a5ad-84c75495ceb3">

**Comment by tangly1024** [comment]
Notion更新了文件上传功能导致， 同另一个bug：https://github.com/tangly1024/NotionNext/issues/1501 ，

**Comment by tangly1024** [comment]
由于Notion接口更新，目前版本暂时不支持PDF在线预览，待后续修复此bug

**Comment by H539251932** [comment]
请问所以现在是修复了吗？
![image](https://github.com/tangly1024/NotionNext/assets/42053563/126ee707-1cb1-4e14-852d-de8c85d45432)
